templates/types/streaming/nextjs/app/components/chat-section.tsx

```diff
@@ -23,6 +23,9 @@ export default function ChatSection() {
   });
 
   const transformedMessages = useMemo(() => {
+    // To test non-streaming response, set NEXT_PUBLIC_CHAT_API=http://localhost:8000/api/chat/request
```

Comment:
probably not bad adding non-streaming support, but how about making this configurable?

e.g.
setting `NEXT_PUBLIC_CHAT_API` for endpoint and
`NEXT_PUBLIC_CHAT_API_DISABLE_STREAMING` for activating non-streaming